server/CONTRIBUTING.md

```diff
@@ -112,10 +111,251 @@ pytest --hge-urls http://127.0.0.1:8080 --pg-urls <database_url> -vv
 - Create a pull request from your forked repo to the main repo.
 - Every pull request will automatically build and run the tests.
 
+
 ## Code conventions
 
 This helps enforce a uniform style for all committers.
 
 - Compiler warnings are turned on, make sure your code has no warnings.
-- Use [hlint](https://github.com/ndmitchell/hlint) to make sure your code has no warnings.
-- Use [stylish-haskell](https://github.com/jaspervdj/stylish-haskell) to format your code.
+- Use [hlint](https://github.com/ndmitchell/hlint) to make sure your code has no
```

Comment:
If `hlint` and `stylish-haskell` are to be part of the official project style it would be good to integrate them into the test suite. That way we avoid muddying up commit history with superficial changes when style violations inevitably get committed.

Reply:
Cool! Sounds like a good idea! I'll create an issue for this so we can track it.

Reply:
https://github.com/hasura/graphql-engine/issues/2490